src/v/config/configuration.cc

```diff
@@ -3697,6 +3697,23 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::chrono::milliseconds(1min),
       {.min = 10ms})
+  , iceberg_catalog_commit_interval_ms(
+      *this,
+      "iceberg_catalog_commit_interval_ms",
+      "How often the Iceberg coordinator commits topic files to the catalog. "
```

Comment:
```suggestion
      "The frequency at which the Iceberg coordinator commits topic files to the catalog. "
```

Reply:
I've made this change, but minor nitpick: the commit interval isn't exactly a frequency (time units), but I think it's not a big deal in common usage.